File: PHP/Car/Car.php
<?php
require_once('Account.php');
class Car {
    public $id;
    public $license;
    public $driver;
    public $passagers;

    public function __construct ($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar(){
        echo '<p>Car License: ' . $this->license . '</p>';
        echo '<p>Driver: '. $this->driver->name .' </p>';
        echo '<p>Driver Document: '. $this->driver->document .' </p>';
    }
}
?>

File: Java/Car/Car.java
package Java.Car;

import Java.AccountType.Driver;;

public class Car {
    Integer id;
    public String license;
    public Driver driver;
    public Integer passagers;

    public Car(String license, Driver driver) {
        this.license = license;
        this.driver = driver;
    }

    public void printDataCar(){
        System.out.println("License: " + license + " Name Driver: " + driver.name);
    }

}
